src/CallableHandler.php

```diff
@@ -122,4 +75,13 @@ public static function execute($callable, array $arguments = [])
             throw $exception;
         }
     }
+
+    protected static function getResolver()
```

Comment:
I am a little bit confused: what is your proposal then? Create a new instance every time?